isso/__init__.py

```diff
@@ -109,16 +109,22 @@ def __init__(self, conf):
 
         subscribers = []
         smtp_backend = False
+        ntfy_backend = False
         for backend in conf.getlist("general", "notify"):
             if backend == "stdout":
                 subscribers.append(Stdout(self))
             elif backend in ("smtp", "SMTP"):
                 smtp_backend = True
+            elif backend == "ntfy":
+                ntfy_backend = True
             else:
                 logger.warning("unknown notification backend '%s'", backend)
         if smtp_backend or conf.getboolean("general", "reply-notifications"):
             subscribers.append(SMTP(self))
 
+        if ntfy_backend:
```

Comment:
No need to introduce a `ntfy_backend` variable, you can just move this block below `elif backend == "ntfy"`